tests/e2e/e2e_setup_test.go

```diff
@@ -26,25 +26,37 @@ import (
 )
 
 const (
-	osmoBalance  = 200000000000
-	osmoDenom    = "uosmo"
-	stakeBalance = 110000000000
-	stakeAmount  = 100000000000
-	stakeDenom   = "stake"
-	minGasPrice  = "0.00001"
+	// common
+	osmoDenom   = "uosmo"
+	stakeDenom  = "stake"
+	minGasPrice = "0.00001"
+	// chainA
+	chainAName    = "osmo-test-a"
```

Comment:
I think we should be more consistent about using ID for referring to chainID's rather than names

Reply:
No problem, on my next PR I will change this to chainAID and so on!